docs/style.md

```diff
@@ -0,0 +1,213 @@
+# InSpec style guide
+
+This is a set of recommended InSpec rules you should use when writing controls.
+
+## Control files
+
+### 1. All controls should be located in the "controls" directory and end in ".rb"
+
+Reason: Most syntax highlighters will render InSpec files correctly across a wide list of tools.
+
+Avoid: `controls/ssh_config`
+Use: `controls/ssh_config.rb`
+
+Avoid: `controls/ssh/config.rb`
+Use: `controls/ssh_config.rb`
+
+### 2. Avoid using "controls" or "control" in the name of your control files
+
+Reason: Using `controls` in the filename again duplicates it and creates unnecessary clutter when reading it. Keep the names short and concise.
+
+Avoid: `controls/ssh_controls.rb`
+Use: `controls/ssh.rb`
+
+
```

Comment:
Suggestion: 
Recommend that control authors do not put line breaks or indenting into the `desc` string.

Reason: Software that executes Inspec and presents the results needs to be able to render controls and results in a UI. Therefore the description for a control should not include line breaks or indenting for the description string that might conflict with formatting later in the UI.

e.g.

Avoid: 

```
desc <-- DESC
    Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
    Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.
    Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur.
    Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
DESC
```

Use:
desc 
```
desc 'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.'
```

Or use Ruby string separation to make a long string more readable in your text editor:

```
desc 'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. '\
'Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. '\
'Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. '\
'Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.'
```

Reply:
That sounds good!